spec/spec_helper.cr

```diff
@@ -4,7 +4,7 @@ ENV["AMBER_ENV"] = "test"
 TEST_PATH     = "spec/support/sample"
 PUBLIC_PATH   = TEST_PATH + "/public"
 VIEWS_PATH    = TEST_PATH + "/views"
-TEST_APP_NAME = "test_app"
+TEST_APP_NAME = "sample-app"
```

Comment:
Two reasons, though this might not be the best place for it:
1. In order to test this fix, the app name needs a dash (-) in it
1. [This test](https://github.com/marksiemers/amber/blob/14ff578d33025fc12bfd06f46d3b81ba9969b610/spec/amber/cli/commands/init_spec.cr#L42) would be a false green for an app containing "test"